packages/app/src/debug/DebugSpec.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div
+    data-cy="debug-spec-col"
+    class="grid flex flex-col px-24px gap-24px self-stretch"
+  >
+    <div
+      data-cy="debug-spec-item"
+      class="w-full overflow-hidden flex flex-col items-start box-border border-1px rounded"
+    >
+      <ul
+        data-cy="debug-spec-header"
+        class="w-full flex flex-row items-center py-12px bg-gray-50 border-b-1px border-b-gray-100 rounded-t"
+      >
+        <li
+          data-cy="spec-contents"
+          class="flex flex-row px-16px items-center gap-x-2"
+        >
+          <span
+            data-cy="spec-path"
+            class="w-145 non-italic text-base"
+          >
+            <span
+              class="font-normal text-gray-600"
+            >
+              {{ specData.initialPath }}
+            </span>
+            <span
+              class="font-medium text-gray-900"
+            >
+              {{ specData.specName.split('.')[0] }}
+            </span>
+            <span
+              class="font-normal text-gray-600"
+            >
+              {{ "." + specData.specName.split('.').slice(1, specData.specName.length).join('.') }}
+            </span>
+          </span>
+          <button
+            v-if="!disabled"
+            data-cy="run-failures"
+            class="inline-flex gap-x-10px whitespace-nowrap h-8 justify-center items-center
+            isolate rounded border border-gray-100 bg-white non-italic text-sm text-indigo-500 font-medium px-12px"
+          >
```

Comment:
Most of our buttons have hover behaviors that either add the border or a background color, but I don't see that here

Reply:
Also, this button should route to the runner to run the selected spec just like clicking the spec in the SpecList page.  Happy to pair on this if that would he helpful.

Reply:
The use of the `Button` component from `frontend-shared` as well as the "wiring" of the button to run the spec has been added on the latest commit